### PR TITLE
Changes CONTRIBUTING from md to rst

### LINKED ISSUE #888: Fix CONTRIBUTING.md
https://github.com/robotframework/SeleniumLibrary/blob/master/CONTRIBUTING.md i outdated. Taka a look on http://www.contribution-guide.org/ and decide how it can be used. 
- [x] Convert to reST. 
- [x] Update doc

### PR DESCRIPTION
**Pull request by aaltat**
Contains also rewording of some parts of the documentation

Fixes #888

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+-  When referring to arguments, argument names must use in inline code
+   style created with double backticks like \`\`argument\`\`.
+
+-  Examples are recommended whenever the new keyword or enhanced
+   functionality is not trivial.
+
+-  All new enhancements or changes should have a note telling when the
+   change was introduced. Often adding something like
+   ``New in Selenium2Libray 1.8.`` is enough.
```
Comment (HelioGuilherme66) [inline CONTRIBUTING.rst]: Old library name.

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+Keyword documentation can be easily created using `invoke`_ task::
+
+    inv generate-documentation
```
Comment (HelioGuilherme66) [inline CONTRIBUTING.rst]: I prefer `invoke generate-documentation`

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+Continuous integration
+----------------------
+
+SeleniumLibrary's continuous integration (CI) servers are visible through
+`Travis CI`_. They automatically test all new pull request to the repository with
+Chrome on Python 2.7, Python 3.3 and Python 3.6. Acceptance test are also run
+by using Selenium 2.53.6 and latest available Selenium 3 version. The used
+Robot Framework versions are: 2.8.7, 2.9.2 and 3.0.2.
+
```
Comment (HelioGuilherme66) [inline CONTRIBUTING.rst]: Could add Firefox (after PR #890).

File: CONTRIBUTING.rst (hunk 1 above)
Comment (aaltat) [reply CONTRIBUTING.rst]: Done

File: CONTRIBUTING.rst (hunk 2 above)
Comment (aaltat) [reply CONTRIBUTING.rst]: Changed to `keyword_documentation`

File: CONTRIBUTING.rst (hunk 3 above)
Comment (aaltat) [reply CONTRIBUTING.rst]: Done

**Pull request merged**